### PR TITLE
ice 3.6.0

### LINKED ISSUE #41572: Ice fails to install with java support
/cc @externl 

Hi Joe,

Sorry to be a pain, but I noticed a problem with the formula after it got merged:

```
% brew install ice --with-java
==> Downloading https://github.com/zeroc-ice/ice/archive/v3.6.0.tar.gz
Already downloaded: /Library/Caches/Homebrew/ice-3.6.0.tar.gz
==> Downloading http://download.oracle.com/berkeley-db/db-5.3.28.NC.tar.gz
Already downloaded: /Library/Caches/Homebrew/ice--berkeley-db-5.3.28.NC.tar.gz
==> ../dist/configure --disable-debug --prefix=/usr/local/Cellar/ice/3.6.0/libexec --mandir=/usr/local/Cellar/ice/3.6.0/libexec/man --enable-cxx --enable-java
==> make install
Error: inreplace failed
cpp/src/slice2freezej/Makefile:
  expected replacement of "${DESTDIR}${binDir}/${appName}.app" with "${prefix}/${appName}.app"
```

This is on yosemite with Java 8 and 7 available. It was certainly working when I tested it previously, but I might not have used the `--with-java` option for my last test before it got merged.

Regards,
Roger

### PR DESCRIPTION
**Pull request by externl**
Fixes --with-java build

Closes #41572

Not sure if I formatted the commit message correctly.

### CODE REVIEW

File: Library/Formula/ice.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Ice < Formula
   homepage "https://zeroc.com"
   url "https://github.com/zeroc-ice/ice/archive/v3.6.0.tar.gz"
   sha256 "77933580cdc7fade0ebfce517935819e9eef5fc6b9e3f4143b07404daf54e25e"
+  revision 1
```
Comment (mikemcquaid) [inline Library/Formula/ice.rb]: Don't need this, thanks.

**Comment by mikemcquaid** [comment]
Preferred commit message is something like `ice: fix --with-java build.`

File: Library/Formula/ice.rb (hunk 1 above)
Comment (externl) [inline Library/Formula/ice.rb]: Ok, I'll remove it.

**Comment by externl** [comment]
Ok, thanks @mikemcquaid!

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**